tests/testthat/test-Rules-methods.R

```diff
@@ -329,3 +329,18 @@ test_that("StoppingLowestDoseHSRBeta works correctly if first active dose is not
   )
   expect_identical(result, expected) # First active dose not applied.
 })
+
+test_that("NextBestInfTheory can be initiliazed ", {
+  # set up
+  newMyNextBest <- NextBestInfTheory(target = 0.25, asymmetry=0.1)
```

Comment:
```suggestion
  new_my_next_best <- NextBestInfTheory(target = 0.25, asymmetry = 0.1)
```

Reply:
oh! that's what it didn't like, I got so hung up on the brackets I completely missed this, will fix.